kubernetes/schema_pod_spec.go

```diff
@@ -355,6 +355,42 @@ func volumeSchema() *schema.Resource {
 		MaxItems:    1,
 		Elem: &schema.Resource{
 			Schema: map[string]*schema.Schema{
+				"default_mode": {
+					Type:         schema.TypeInt,
+					Description:  "Optional: mode bits to use on created files by default. Must be a value between 0 and 0777. Defaults to 0644. Directories within the path are not affected by this setting. This might be in conflict with other options that affect the file mode, like fsGroup, and the result can be other mode bits set.",
+					Optional:     true,
+					ValidateFunc: validateModeBits,
```

Comment:
Does it make sense to add `Default: 0644` here? Either for clarity for the user when they see the diff and/or for consistency, otherwise the default value (if undefined) here will be `0` which I'm not sure is the same in this context.